configs/endpoints/aws.yaml

```diff
@@ -20,8 +20,6 @@ worker_node_types:
       "accelerator_type:T4": 1
   min_workers: 0
   max_workers: 100
-  use_spot: true
```

Comment:
hmm. how will customers who want spot instances configure this in template?

Reply:
they can edit the compute config